LibraBFT/Impl/Consensus/RoundManager.agda

```diff
@@ -24,48 +31,126 @@ open import LibraBFT.Abstract.Types.EpochConfig UID NodeId
 
 module LibraBFT.Impl.Consensus.RoundManager where
 
-  open RWST-do
-
-  processCommitM : LedgerInfoWithSignatures → LBFT (List ExecutedBlock)
-  processCommitM finalityProof = pure []
-
-  fakeAuthor : Author
-  fakeAuthor = 0
-
-  fakeBlockInfo : Epoch → Round → ProposalMsg → BlockInfo
-  fakeBlockInfo eid rnd pm = BlockInfo∙new eid rnd (pm ^∙ pmProposal ∙ bId)
-
-  fakeLedgerInfo : BlockInfo → ProposalMsg → LedgerInfo
-  fakeLedgerInfo bi pm = LedgerInfo∙new bi (pm ^∙ pmProposal ∙ bId)
-
-  postulate -- TODO-1: these are temporary scaffolding for the fake implementation
-    fakeSK  : SK
-    fakeSig : Signature
-
-  processProposalMsg : Instant → ProposalMsg → LBFT Unit
```

Comment:
I would like to keep our "fake" implementation (and proofs about it) for some time, rather than replacing them.  I guess we should move them somewhere else so they no longer pollute the namespace of the actual implementation model.  I think if you did that, the result would be the files the @cwjnkins is building on looking the same, thus avoiding conflicts for him, while establishing a new location for the fake handler.  Does this make sense?

Reply:
I understand you'll do this in a subsequent PR.  I thought that your changes would break the existing proofs about the fake handler, which I wouldn't want to merge, but now I realise that you didn't actually hook up the partial implementation to the handler, so no proofs were harmed in the production of this PR :smile:.